packages/client/src/Views/PlayerLog/index.tsx

```diff
@@ -41,11 +44,18 @@ const PlayerLogComponent: React.FC<PlayerLogProps> = ({ isOpen, onClose }): Reac
     }
   }, [isOpen, playerMessageLog])
 
+  const selectedForceId = selectedForce ? selectedForce.uniqid : ''
+  
   const collatePlayerLogData = (messageLog: PlayerMessageLog): void => {
-    getPlayerActivityLogs().then((activityLog: ActivityLogsInterface[]) => {
+    getPlayerActivityLogs(currentWargame, currentDbname).then((activityLog) => {  
```

Comment:
@lilitkarapetyan - this method is "putting" the player activity logs, rather than getting them. Could you rename it please?